### PR TITLE
Lykhoyda/add snap

### PR DESCRIPTION
**Pull request by Lykhoyda**

### CODE REVIEW

File: packages/demo-wallet/dist/_headers
Comment (willemolding) [inline packages/demo-wallet/dist/_headers:1]: This is used by Netlify (and other web hosting platforms) to set the headers correctly. Please put it back 😆

**Review by willemolding — APPROVED** [review]
If you we going to apply as global lint to the demo-wallet ( and we definitely should, I'm glad you added that) we should also include the .eslint config in that package so everyone's editors will respect the same style.

Other than that and the issue with removing the `_headers` file this looks good to merge

**Comment by willemolding** [comment]
Please see https://github.com/Lykhoyda/WebZjs/pull/1/files

**Comment by Lykhoyda** [comment]
> Please see https://github.com/Lykhoyda/WebZjs/pull/1/files

I think it's also a good strategy so far to make it more minimalistic as we don't require any customization since we split the wasm packages. I merged your PR and added some adjustments. Please take a look 👍

**Review by willemolding — APPROVED** [review]

**Pull request merged**